microsoft/iot-cardboard-js
# OAT: Refactor import/export logic to public utils file

### Summary of changes 🔍 
Pulling out the import, export and parsing logic to a public utility file. It doesn't change the functionality, just moves it to a common place so we can iterate and expose to the ADT Explorer as well in the future.

### Testing 🧪
- import a file
- import a folder
- Import a file that already exists in the ontology, see the errors
- Import a file that is not a JSON file
- Import a file that is a JSON file but the contents don't parse properly
- export ontology
